tests/test_consumer.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+import pytest
+
+from alluvium.consumer import KinesisFirehoseConsumer, Wal2JsonProcessorV1, Wal2JsonProcessorV2
+
+
+@pytest.fixture
+def kinesis_delivery_stream():
+    return "foo-stream"
+
+
+class MockLogicalReplicationEvent(object):
+    def __init__(self, consumable):
+        self.payload = consumable
+
+
+def test_wal_2_json_processor_v1():
+    cdc_event = json.dumps({"change": [{"foo": "bar"}]})
+
+    processor = Wal2JsonProcessorV1(cdc_event)
+    assert processor.should_keep
+    assert processor.post_process() == {"change": [{"foo": "bar"}]}
+
+    cdc_event = json.dumps({"change": []})
+    processor = Wal2JsonProcessorV1(cdc_event)
+    assert not processor.should_keep
+
+
+def test_wal_2_json_processor_v2():
+    cdc_event = json.dumps({"action": "I", "foo": "bar"})
+    processor = Wal2JsonProcessorV2(cdc_event)
+    assert processor.should_keep
+    assert processor.post_process() == {"action": "I", "foo": "bar"}
+
+    cdc_event = json.dumps({"action": "B", "foo": "bar"})
+    processor = Wal2JsonProcessorV2(cdc_event)
+    assert not processor.should_keep
+
+
+def test_kinesis_consumer(mocker, kinesis_delivery_stream):
```

Comment:
call out for @codehruv. Notice how I am just checking mock calls, this isn't testing anything very useful and I would argue is super brittle because these tests will break as soon as an interface changes. Your issue would make testing a lot more useful.